style.css

```diff
@@ -0,0 +1,72 @@
+.logo{
+    text-decoration: underline;
+    text-decoration-color: var(--primary-color);
+    padding-bottom: 0.5rem;
+}
+
+nav{
+    align-items: center;
+}
+
+.main{
+    min-height: 70vh;
+    align-items: center;
+}
+
+.content-center{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.img{
+    border-radius: 0.5rem;
+}
+
+.categories-container{
+    display: flex;
+    justify-content: center;
+    gap: 1rem;
+    margin: auto;
+}
+
+.flex-space-between{
+    display: flex;
+    justify-content: space-between;
+}
+
+.container{
+    max-width: 45rem;
+    min-height: 70vh;
+    margin: auto;
+    padding: 2rem;
+}
+
+.options-container{
+    display: flex;
```

Comment:
you can create a flex layout css and have all the classes there and then just keep it in html. This will make it very easy to change and it helped me. This is just a suggestion you can implement it later.

Reply:
Sure, will implement this